content/projects/microbial/index.md

```diff
@@ -0,0 +1,137 @@
+---
+description: "Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics transcriptomics or proteomics, this is the place to be!"
+autotoc: true
+title: "Microbial data analysis in Galaxy"
+---
+
+Whether you are analysing microbiome samples or bacterial isolates, long reads or short, shotgun or 16S, genomics transcriptomics or proteomics, **this is the place to be**!
+
+**[microGalaxy](#microgalaxy-community)** is a community of practice in the Galaxy community for anything regarding microbial data analysis in Galaxy!
+
+
+# What is available?
+
+
+## Tools
+
+**Many tools for microbial data analysis are freely available** in the ToolShed which can be installed on any Galaxy server. To keep an overview, the *[microGalaxy](#microgalaxy-community)* community of practice maintains a [**curated list**](https://docs.google.com/spreadsheets/d/1Nq_g-CPc8t_eC4M1NAS9XFJDflA7yE3b9hfSg3zu9L4/edit#gid=101894501) of the **150+ tools** available.
+
+If tools are missing or information is not up-to-date in the list, please help us! Contact Saskia or Bérénice about it.
```

Comment:
Can we add Paul Zierep here, and maybe email links ? 
Does it make sense to extract the available tools and add somewhere at the homepage ...?